static/index.scss

```diff
@@ -48,55 +48,88 @@
     padding: 0 .5em;
     border-right: 4px solid black;
 
+    &__form {
+      font-family: $title;
+      font-style: italic;
+      background-color: black;
+      color: white;
+      border: none;
+      border-radius: 0.5em;
+      text-align: center;
+      font-size: 1.5em;
+      width: 100%;
+      padding-top: 0.25em;
+      padding-bottom: 0.25em;
+      margin-top: 0em;
+      margin-bottom: 1em;
```

Comment:
Condense these:
```css
padding: 0.25em 0;
margin-bottom: 1em;
```
If you just put two values in `padding`, it will use the first as top and bottom, and the second as left and right.